helloworld-simple-project/README.md

```diff
@@ -0,0 +1,60 @@
+# helloworld-simple-project Project
```

Comment:
Please change the name of the top level folder to hello-quarkus and update this README to actually build, test and deploy the function - both locally as well as in the cluster using kn func emit etc

Reply:
@rsriniva I have updated the readme file as per the mandatory process required to successfully run the serverless quarkus functions.